packages/react-scripts/fixtures/kitchensink/README.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to Create React App's E2E tests
+
+This test suite is a [kitchensink](FIXME: link needed), but has multiple usages in it.
+
+## Running the test suite
+
+Tests are automatically run by the CI tools.  
+In order to run them locally, without having to manually install and configure everything, the `yarn e2e:docker` CLI command can be used.
+
+This is a simple script that runs a **Docker** container, where the node version, git branch to clone, test suite, and whether to run it with `yarn` or `npm` can be chosen.  
+Simply run `yarn e2e:docker -- --help` to get additional info.
+
+N.B.: this command will clone the local repository in the docker container, so any uncommitted changes are ignored.
```

Comment:
Is there a way to run this on uncommitted changes. As normally I'd make changes, write tests, run tests. And then if everything works commit changes.

With this, I'd be making changes, committing, testing, making changes, committing, testing, etc.